public/css/login.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
```

Comment:
why the width is 100%! if you remove it, this will not affect anything :smile:

Reply:
:+1: